content/post/087-deprecating-the-old-docs-site.md

```diff
@@ -0,0 +1,53 @@
+---
+date: 2020-04-01
+url: 2020-04-01-deprecating-the-old-docs-site
+title: Deprecating the old docs site
+author: Johnny Matthews
+---
+
+# Goodbye, IPFS legacy docs
+
+The IPFS team is deprecating the old documentation website. That’s just a fancy way of saying we’re replacing the current docs.ipfs.io website with the new docs-beta.ipfs.io website.
+
+## We built a new docs site
+
+Work on a new documentation site for IPFS started in Q3 of 2019. Before that, the docs site was built upon Hugo. While Hugo is an excellent static-site generator, it lacks an encapsulated component model, relies upon old web development conventions, and requires DOM manipulation if you want to have any kind of site interactivity.
```

Comment:
Is any part of "relies upon old web development conventions" a bit too negative to mention them by name?

Reply:
Not that I know of, but I didn't want to dive too deep into this. There's already out in the wild about Hugo's issues.

Reply:
I don't think these details are being helpful here. "encapsulated component model" / "old web developemnt conventions" / "DOM manipulation" are all jargon and not related to the user's actual question which is "what part of my docs usage experience will be better now?" Suggest we reframe around that context.

```suggestion
A common complaint from users new and old was that the IPFS onboarding experience was rough - and that the documentation was particularly difficult to navigate to find fast, reliable answers. We set out to change that! Work on a new documentation site for IPFS started in Q3 of 2019. Before that, the docs site was built upon Hugo. While Hugo is an excellent static-site generator, it was missing some features around interactive content that we needed to adaptively respond to user needs and collect feedback on which pages to improve first.
```

Reply:
Yeah that works better.